planning/behavior_path_planner/include/behavior_path_planner/util/drivable_area_expansion/parameters.hpp

```diff
@@ -0,0 +1,111 @@
+// Copyright 2022 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef BEHAVIOR_PATH_PLANNER__UTIL__DRIVABLE_AREA_EXPANSION__PARAMETERS_HPP_
+#define BEHAVIOR_PATH_PLANNER__UTIL__DRIVABLE_AREA_EXPANSION__PARAMETERS_HPP_
+
+#include "behavior_path_planner/util/drivable_area_expansion/types.hpp"
+
+#include <rclcpp/node.hpp>
+#include <vehicle_info_util/vehicle_info_util.hpp>
+
+#include <string>
+#include <vector>
+
+namespace drivable_area_expansion
+{
+
+struct DrivableAreaExpansionParameters
+{
+  static constexpr auto ENABLED_PARAM = "dynamic_expansion.enabled";
```

Comment:
Can I ask why you hold a parameter name as a member variable.

Reply:
So I do not have to write the parameter twice (declaring and updating)
https://github.com/autowarefoundation/autoware.universe/blob/f1d53ddc66efe35cd61c3f2bf710dec39c13dc70/planning/behavior_path_planner/src/behavior_path_planner_node.cpp#L1100-L1102

Reply:
Got it. Thanks.